src/js/pubsub.js

```diff
@@ -2,7 +2,12 @@ import fire from './fire';
 import on from './on';
 import debounce from './debounce';
 
-const cache = {};
+// @TODO: this local variable isn't shared between redundant module instance
```

Comment:
@LucaMele 
I have tracked this issue in #44

Reply:
Hello Andy, thank you. I dont think thought that local variables should be shared as its a closed module/component. If we would have shadow dom, we would have the same thing.

Reply:
It have to be thourgh an external module/window

Reply:
we need to invent a mechanism for this. I consider it as a feature.

Reply:
Best having like a global module. Similar how the app.js works, just not for the SG but for the WCs

Reply:
@LucaMele 
I'm not sure if we talk about the same thing.
We have module **A** with a local variable `localVar`.

If that module is bundled redundantly into various components, instead of a shared `vendor` bundle, then multiple instance of module **A** exist, each with their own scope and own local variables, without access to each other.

Reply:
Exactly. This is not as issue but just a limitation of having completely self contained modules. It makes absolutely sense as a single module should work always on its own (holistic component) and can be integrated in a System over connections. 

The concept is different from having one bundle with everything. Here we have self contained components. That means, no sharing.

We need to have a kind of vendor module/file where other modules can share properties. This would be a nice feature. We need to discuss it.

Reply:
So all those vars should be living in one external file so they wont be multiple bundled when integrating them in a bundled system

Reply:
and that system should be avaiable inside the components. Hope it make sense what im saying.